src/anchoredinterval.jl

```diff
@@ -121,6 +121,12 @@ end
 
 ##### ACCESSORS #####
 
+# Note: For `first` and `last` the typically we would compute this is using `min` and `max`.
```

Comment:
"We would typically compute `first` and `last` using `min` and `max` respectively, but we can get unexpected behaviour if adding the span to the endpoint produces a value that is no longer comparable (e.g., `NaN`)."?